Introduced/used the phrase processed SD-JWT payload in Section 8.1 on Verifying the SD-JWT (#456)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

5. If any Disclosure was not referenced by digest value in the Issuer-signed JWT (directly or recursively via other Disclosures), the SD-JWT MUST be rejected.
6. Check that the SD-JWT is valid using claims such as `nbf`, `iat`, and `exp` in the processed payload. If a required validity-controlling claim is missing (see (#sd-validity-claims)), the SD-JWT MUST be rejected.

> If any step fails, the SD-JWT is not valid, and processing MUST be aborted. Otherwise, the JSON document resulting from the preceding processing and verification steps, henceforth referred to as the processed SD-JWT payload, can be made available to the application to be used for its intended purpose.

Author: with #457 the term will used a bit more ...

```suggestion
If any step fails, the SD-JWT is not valid, and processing MUST be aborted. Otherwise, the JSON document resulting from the preceding processing and verification steps, herein referred to as the processed SD-JWT payload, can be made available to the application to be used for its intended purpose.
```

   -12

   * Editorial and reference fixes

>    * Introduced the phrase processed SD-JWT payload in the end of the (#sd_jwt_verification) on Verifying the SD-JWT

Author: ```suggestion
   * Introduced the phrase processed SD-JWT payload in the end of (#sd_jwt_verification) on Verifying the SD-JWT
```

   -12

   * Editorial and reference fixes

>    * Introduced the phrase processed SD-JWT payload in the end of (#sd_jwt_verification) on Verifying the SD-JWT

Author: ```suggestion
   * Better introduce the phrase processed SD-JWT payload in the end of (#sd_jwt_verification) on Verifying the SD-JWT
```
